Shared/Item/ItemDestinationView.swift

```diff
@@ -35,7 +41,9 @@ struct ItemDestinationView: View {
         destinationView
             .toolbar(item: item,
```

Comment:
Another bit of style commentary.

I feel like there are two basic styles to writing method calls: I'll call them 1) single-line and 2) chopped-down.

The formatting for the single-line style is pretty self-evident so I won't go into that. As for the chopped-down version, here's what I like:

1. First parameter goes on a new line
2. Each parameter goes on its own line
2. Closing paren is on its own line

For example:

```swift
destinationView.toolbar( // no param here
    item: item, // first param goes on a new line
    presentationMode: ...,
    shouldPresentWebView: ...
).anotherMethod() 
// closing paren on its own line, easier to see what the chained method is attached to
```

I particularly like this over the format of keeping the first param on the same line as the opening paren because it avoid large swaths of whitespace before your eyes get to the next param

```swift
destinationView.toolbar(item: item, // first param on same line as opening paren
                        presentationMode: ..., // now there is a huge crater of whitespace before the second param
                        shouldPresentWebView: ...) // closing paren here is hard for my eyes to see the end of the statement
    .anotherMethod() // now this chained method is kinda out on its own
```

Reply:
I like this! Does this also come into play when defining a function, or primarily at the call site?

Reply:
Yes, same rules apply for method definitions as well 👍